spec/client_spec.rb

```diff
@@ -367,6 +367,36 @@ def redis.with
     end
   end
 
+  describe 'get_remaining_ttl' do
+    after(:each) { lock_manager.unlock(@lock_info) if @lock_info }
+
+    it 'gets the remaining ttl of a lock' do
+      ttl = 20_000
+      @lock_info = lock_manager.lock(resource_key, ttl)
+      remaining_ttl = lock_manager.get_remaining_ttl(@lock_info)
+      expect(remaining_ttl).to be_within(300).of(ttl)
+    end
+  end
```

Comment:
* returns `nil` when lock is not held
* test that it returns minimum somehow, probably by mocking redis away